apps/e2e-tests/src/test-scenarios/wit/expected-service-status.ts

```diff
@@ -0,0 +1,46 @@
+export const expectedPCStatuses03 = {
```

Comment:
I think this file is more or less the same as `apps/e2e-tests/src/test-scenarios/e2e-general/expected-service-status.ts`. Can you import the statuses from there and re-`export` them here?